spinoso-time/src/time/tzrs/timezone.rs

```diff
@@ -0,0 +1,155 @@
+use tz::datetime::DateTime;
+
+use super::{Offset, Time};
+
+// Timezone conversions (returns new Time)
```

Comment:
thank you thank you a million times for figuring out offsets and timezones!

Reply:
I'm confident that my hair will eventually be grey later in life - and I think offset/timezones definitely got me a little closer to that haha.